internal/controller/cluster_monitoring_controller.go

```diff
@@ -122,8 +125,14 @@ func (r *ClusterMonitoringReconciler) reconcile(ctx context.Context, cluster *cl
 		}
 	}
 
+	err := r.MimirService.ConfigureMimir(ctx, r.ManagementCluster.Name)
```

Comment:
Do we need it to run for all clusters? Maybe we could run this only for the MC reconciliation?

Reply:
True